interface/main/calendar/add_edit_event.php

```diff
@@ -89,6 +89,11 @@
  $info_msg = "";
 
  ?>
+
+<?php $edit = acl_check("groups","gcalendar",false, 'write');?>
+<?php $view = acl_check("groups","gcalendar",false, 'view');?>
```

Comment:
I am sorry I didn't note this before, but need to make clear these variables are for groups stuff or else will make an already confusing script more confusing.
So change $edit to $g_edit and change $view to $g_view .